python-sdk/src/astro/databricks/databricks_api_utils.py

```diff
@@ -0,0 +1,144 @@
+import fileinput
```

Comment:
Since this module is already in the `databricks` package, perhaps we could only name it `api` or give a more specific name as `delta`, if the classes/methods are specific to delta.

Reply:
+1